content/en/Platform Deep Dive/Pentests/Findings/remediate-findings.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Review and Take Action on Findings"
+linkTitle: "Take Action on Findings"
+weight: 20
+description: >
+  Remediate or accept findings.
+---
+
+{{% pageinfo %}}
+During remediation, review findings that our pentesters discovered, and take action on them.
+{{% /pageinfo %}}
+
+Once our pentesters move a finding to [Pending Fix](/platform-deep-dive/pentests/findings/finding-states/), you can:
+
+- Fix the finding and [submit it for retest](#submit-a-finding-for-retest)
+- [Mark the finding as Accepted Risk](#mark-a-finding-as-accepted-risk)
+
+Our pentesters describe findings and provide recommendations on how to fix them. Navigate to the finding page for details.
+
+## Submit a Finding for Retest
+
+Once you've fixed a finding internally, you can submit it for retest.
+
+1. On the **Findings** tab of the pentest page, filter **Pending Fix** findings.<br><br>
+![Filter Pending Fix findings](/deepdive/FilterPendingFixFindings.png "Filter Pending Fix findings")
+1. Select the desired finding.
+1. In the **State** list, select **Ready for Retest**.<br><br>
+![Submit a finding for retest](/deepdive/ReadyForRetestFinding.png "Submit a finding for retest")
+
+The pentester who posted the finding gets notified and retests the issue. When finished, they change the [state](/platform-deep-dive/pentests/findings/finding-states/) to:
+
+- Fixed, if they can't reproduce the issue.
+- Pending Fix, if the issue persists. Read the pentester's comment for details.
+
+### Free Retesting Duration
```

Comment:
While I think this looks accurate, it bothers me.

- I think this is the kind of detail that belongs in our contract. I'd rather not involve our lawyers in our product docs
- The PtaaS Tier webpage already states the free retesting period

I'm guessing these details are also "subject to negotiation," which could affect the credibility of this section.

Reply:
@mjang-cobalt Please see my comment here: https://github.com/cobalthq/cobalt-product-public-docs/pull/254#issuecomment-1347425095